src/services/storageTypes.ts

```diff
@@ -37,7 +37,7 @@ export function fromTitle(title: string): che.WorkspaceStorageType {
 }
 
 export function getAvailable(settings: che.WorkspaceSettings): che.WorkspaceStorageType[] {
-  const availableTypes = settings['che.workspace.storage.available_types'];
+  const availableTypes = settings['che.workspace.storage.available_types'] || 'persistent,ephemeral,async';
```

Comment:
I don't understand how available types and websockets are related since available types are supposed to be fetched from workspace/setting HTTP GET request, I assume dashboard initialization process is interrupted if WebSockets are not available, but it makes sense to continue it...
Seems something is wrong with this line https://github.com/che-incubator/che-dashboard-next/blob/master/src/services/bootstrap/PreloadData.ts#L49
@akurinnoy @olexii4 WDYT?

BTW It makes sense to assign some default value, even `''` is better than `undefined`, to avoid dashboard crashing.

Reply:
I totally agree(maybe without 'async').

Reply:
BTW Here is issue that Dashboard should display error instead of blank page when it fails to initialize https://github.com/eclipse/che/issues/18758